apps/etterlatte-behandling/src/main/kotlin/behandling/BehandlingFactory.kt

```diff
@@ -360,6 +362,7 @@ class BehandlingFactory(
                         oppgaveKilde = OppgaveKilde.BEHANDLING,
                         merknad = merknad,
                         gruppeId = null,
+                        soeknadMottattDato = nyFoerstegangsbehandling.soeknadMottattDato,
```

Comment:
Ingen test som dekker use-caset?

Reply:
På denne 150-linjers funksjonen for å se om vi sender med et parameter? nei jeg gadd ikke :p

kan se om jeg kan tilpasse en av de andre testene evt.

Reply:
[Tester business-logikk for setting av frist](https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/7064/commits/e46ec4bd309b3284080e4255b121642b038796ba)